### PR TITLE
Save generated plots

### PR DESCRIPTION
**Pull request by vnykr**
Hi, 

I have added the option to save the generated plots based on the suggestions in Issue #14. Here are the changes I have made. 

**1. Add argument `fname` to the plotting functions**
If `fname` is specified then simply calls `plt.savefig(fname)` to save the plot result. 

**2. Add argument `save_dir` to `visual_evaluation()` function** 
If `save_dir` is specified then saves all generated plots into this directory. Each image has a fixed name in this case. 

Hope this is useful.

### CODE REVIEW

**Comment by Baukebrenninkmeijer** [comment]
@vnykr Looks great! 

Could you make develop the target? That's the branch where it should be merged first, since I'm following a gitflow working method. Thanks!

File: table_evaluator/table_evaluator.py [hunk 1]
```diff
@@ -354,17 +372,28 @@ def score_estimators(self):
...
+            self.plot_mean_std()
+            self.plot_cumsums()
+            self.plot_distributions()
+            self.plot_correlation_difference(**kwargs)
+            self.plot_pca()    
+        else: 
+            if not os.path.isdir(save_dir):
+                os.makedirs(save_dir)
+
+            self.plot_mean_std(fname=os.path.join(save_dir, 'mean_std.png'))
```
Comment (Baukebrenninkmeijer) [inline table_evaluator/table_evaluator.py:391]: Could you replace these `os.path.join` with 
```python
from pathlib import Path
save_dir = Path(save_dir)

self.plot_mean_std(fname=save_dir/'mean_std.png'
```

and the same for the other paths. I prefer working with the path library over os.path.join.

**Review by Baukebrenninkmeijer — CHANGES_REQUESTED** [review]
Just those filenames and I'm good. Please make sure to target develop :).

**Comment by vnykr** [comment]
@Baukebrenninkmeijer My apologies for targeting the wrong branch, this is my first time making a pull request! 
I think I have made the correct changes, please let me know if there is anything else.

**Pull request merged**